packages/mendel-browserify/index.js

```diff
@@ -203,6 +210,35 @@ MendelBrowserify.prototype.doneManifest = function() {
     );
 }
 
+MendelBrowserify.prototype.sortedManifest = function() {
+    var sortedManifest = {
+        indexes: {},
+        bundles: []
+    };
+
+    var self = this;
+    Object.keys(this._manifestIndexes).sort().forEach(function(file) {
+        var bundle = self._manifestBundles[self._manifestIndexes[file]];
+        sortedManifest.bundles.push(bundle);
+
+        var index = sortedManifest.bundles.indexOf(bundle);
```

Comment:
Oh! Sorry, I probably misread your line when I first read.

Yeah, I could do that too. And could also have used the second argument of the forEach funcion for index. Since I have a similar code during manifest generation — and there we have potential racing conditions — I chose to make it look like the other line.

There is a bit code swag there 😄:
It feels natural "add the bundle to the array, retrieve the bundle position, and store the index under the file key" feels better than "add the bundle to the array, get the position for the last item in the array, , and store the index under the file key".